cmd/prometheus/config.go

```diff
@@ -190,45 +187,6 @@ func init() {
 		"Local storage engine. Supported values are: 'persisted' (full local storage with on-disk persistence) and 'none' (no local storage).",
 	)
 
-	// Remote storage.
-	cfg.fs.StringVar(
```

Comment:
I would have assumed that flags are part of the code that implements those features. Isn't it more dangerous to remove the flags and silently break things for users rather than obviously break it? There's an open thread about that in https://docs.google.com/document/d/1TFhL6kEasDxkk3pxsHcM6TqSx4E4NI3X2kshNNQ6-X0/edit where you seem to have formulated a similar position? "Do we want those tweaks to be ignored, or make sure users know when the flag disappears and their binary doesn't start when they try to upgrade? I'm generally not a big fan of silent breakage."